src/components/Post/ActionsBar/ReportButton/index.tsx

```
@@ -164,19 +172,60 @@ const ReportButton: FC<IReportButtonProps> = (props) => {
+				postType: proposalType
+			});
+
+			if (error) {
+				queueNotification({
+					header: 'Error!',
+					message: 'Error deleting post',
+					status: NotificationStatus.ERROR
+				});
+			}
+
+			if (data && data?.message) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)